### PR TITLE
Add services metrics and split dashboards in 2 more specialized dashboard

### PR DESCRIPTION
**Pull request by MalloZup**
# Description:

This pr split the dashboard in 2 parts. 

- 1) is a real-time dashboard mainly to know the cluster status etc as alerting
- 2)  will focus more on real-time analsys of metric on timestamps. (for moment it is more the old one)

# Real-time dashboard

dash-cluster-status

### CODE REVIEW

**Comment by MalloZup** [comment]
alerts

**Comment by MalloZup** [comment]
alerts

**Comment by MalloZup** [comment]
the `detailed` dashboard right now is the old hana sap-db dashboard.

**Comment by MalloZup** [comment]
image

File: salt/monitoring/prometheus/rules.yml [hunk 1]
```diff
@@ -1,28 +1,83 @@
...
+  - alert: sap-hana-master-resource-down
     expr: absent(cluster_node_resources{resource_name="rsc_saphana_prd_hdb00",role="master"})
     labels:
       severity: page
     annotations:
       summary: Primary SAP-HANA resource down
 
-- name: a-resource-failed
+  - alert: sap-hana-secondary-resource-absent
+    expr: absent(cluster_node_resources{resource_name="rsc_saphana_prd_hdb00",role="slave",status="active"})
```
Comment (diegoakechi) [inline salt/monitoring/prometheus/rules.yml]: @MalloZup This name is dynamic composed, being 00 the InstanceNr, for example. I would make it a generic alert saing "Slave Resource is absent" and if possible, I would include the content of the label `resource_name` into the Alert description.

File: salt/monitoring/prometheus/rules.yml [hunk 2]
```diff
@@ -1,28 +1,83 @@
...
+
+  - alert: service-down-ha-cluster-exporter
+    expr: node_systemd_unit_state{name="prometheus-ha_cluster_exporter.service", state="active"} == 0
+    labels:
+      severity: page
+    annotations:
+      summary: ha cluster exporter service not running
+ 
+  - alert: service-down-hanadb-exporter
+    expr: node_systemd_unit_state{name="hanadb_exporter@prd_00.service", state="active"} == 0
```
Comment (diegoakechi) [inline salt/monitoring/prometheus/rules.yml]: @MalloZup same here... the `prd_00` are dynamic related to the SID and InstanceNR. Can we make it like a contains? if any hanadb_exporter service fails, an Alert should be raised...

File: salt/monitoring/prometheus/rules.yml (hunk 1 above)
Comment (MalloZup) [reply salt/monitoring/prometheus/rules.yml]: done

File: salt/monitoring/prometheus/rules.yml (hunk 2 above)
Comment (MalloZup) [reply salt/monitoring/prometheus/rules.yml]: dpme

File: salt/monitoring/prometheus/rules.yml (hunk 1 above)
Comment (diegoakechi) [reply salt/monitoring/prometheus/rules.yml]: @MalloZup I don't see the change.... the expression will still fail if the resource name is not rsc_saphana_prd_hdb00.

File: salt/monitoring/prometheus/rules.yml [hunk 3]
```diff
@@ -1,28 +1,83 @@
 groups:
-- name: primary-resource-saphana-down
+- name: sap-hana-resource-monitoring
   rules:
-  - alert: SAPHANAMasterResourcesDown
-    expr: absent(cluster_node_resources{resource_name="rsc_saphana_prd_hdb00",role="master"})
+  - alert: sap-hana-master-resource-down
+    expr: absent(cluster_node_resources{resource_name=~"rsc_saphana_prd_hdb.*",role="master"})
```
Comment (diegoakechi) [inline salt/monitoring/prometheus/rules.yml]: @MalloZup the `prd` part is also dynamic. 

```suggestion
    expr: absent(cluster_node_resources{resource_name=~"rsc_saphana_.*",role="master"})
```

**Comment by MalloZup** [comment]
@diegoakechi thx! :100:

**Pull request merged**